bin/index.js

```diff
@@ -5,37 +5,36 @@ const { existsSync, readFileSync } = require('fs');
 const {
   generateRequestFile,
   generateAirtableFile,
-  generateConstantsFile,
   generateSchemaFile
 } = require('../lib/generators');
 
-// Get Airtable Credentials into environment
-// require('dotenv-safe').config({
-//   path: '.airtable-schema-generator.env',
-//   example: path.resolve(__dirname, '../.env.example')
-// });
-
 const script = `copy(_.mapValues(application.tablesById, table => _.set(_.omit(table, ['sampleRows']),'columns',_.map(table.columns, item =>_.set(item, 'foreignTable', _.get(item, 'foreignTable.id'))))));`;
 
 function readSettings() {
   const packageFile = readFileSync('./package.json');
   const settings = JSON.parse(packageFile)['airtable-schema-generator'];
-  console.log('Found settings');
+  console.log('Found settings:');
   console.log(settings);
 
-  if (!settings || !settings.output || !settings.input || !settings.baseId) {
+  if (!settings || !settings.output || !settings.baseId || !settings.mode) {
     console.log(
-      "Couldn't find Input Folder Path, Output Folder Path and Base ID in Settings Object:"
+      "Couldn't find Input Folder Path, Output Folder Path, Mode, and Base ID in Settings Object:"
```

Comment:
Nitpick: error message shouldn't always include "Input Folder"

Reply:
good catch, fixed in a future PR because i caught it there